Velaptor/Graphics/Renderers/ITextureRenderer.cs

```diff
@@ -96,8 +97,100 @@ public interface ITextureRenderer
     ///         <item>Texture 5</item>
     ///     </list>
     /// </remarks>
+    [SuppressMessage("ReSharper", "UnusedMemberInSuper.Global", Justification = "Public API for users.")]
     void Render(ITexture texture, int x, int y, float angle, int layer = 0);
 
+    /// <summary>
+    /// Renders the given texture at the given <paramref name="x"/> and <paramref name="y"/> coordinates <paramref name="angle"/>,
+    /// and <paramref name="size"/>.
+    /// </summary>
+    /// <param name="texture">The texture to render.</param>
+    /// <param name="x">The X location of the texture.</param>
+    /// <param name="y">The Y location of the texture.</param>
+    /// <param name="angle">The angle of rotation in degrees of the rendering.</param>
+    /// <param name="size">The size to render the texture. 1 is for 100%/normal size.</param>
```

Comment:
Refactor this param doc.

Do this for all copies of this param docs.

```suggestion
    /// <param name="size">The size to render the texture. 1 is for 100%, which is the normal size.</param>
```